docs/how-to/deploy_sdcore_user_plane_with_hugepages.md

```diff
@@ -0,0 +1,64 @@
+# Deploy SD-Core User Plane with HugePages
+
+This guide covers how to deploy the User Plane Function (UPF) with huge pages using 
+the `sdcore-user-plane` Juju bundle.
+
+## Requirements
+
+- A Kubernetes host which meets the following requirements: 
+  - CPU that supports AVX2 and RDRAND and PDPE1GB instructions (Intel Haswell, AMD Excavator or equivalent)
+  - LoadBalancer with 1 available address for the UPF
+  - Multus CNI enabled
+- Juju >= 3.1/stable
+- A Juju controller bootstrapped onto the Kubernetes host
+- Juju model created, named `user-plane`
+
+## Enable HugePages on the host
+As `root` user, edit the bootloader of the OS appending the following command to the
+kernel command-line parameter:
+```console
+default_hugepagesz=1G
+```
+Then update the bootloader.
```

Comment:
I think more details in this section would be good, like what file to edit and how to update the bootloader configuration.

Reply:
I was intentionally not so detailed because the path of the file to edit and command to update the bootloader may depend on the bootloader itself. If we all agree, I can add all the steps for GRUB only. @gruyaume